recipe/meta.yaml

```diff
@@ -17,18 +17,24 @@ source:
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [osx and arm64]
 
 build:
-  number: 0
+  number: 1
+  skip: true  # [win and cuda_compiler_version != "None"]
+  skip: true  # [cuda_compiler_version == "10.2"]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
     - git                    # [not win]
     - m2-git                 # [win]
     - cmake
     - llvm-openmp  # [osx]
     - ninja
   host:
+    - cudatoolkit      # [cuda_compiler_version != "None"]
+    - cudnn      # [cuda_compiler_version != "None"]
```

Comment:
I'm not familiar with conda packaging, but cudnn seems to be unnecessary?

Reply:
Removed in https://github.com/conda-forge/xgboost-feedstock/pull/84/commits/83a0714d4a46ab8e236f62fc1efd92176833f251 thank you!